README.md

```diff
@@ -48,7 +49,33 @@ simplistika.css is a lightweight, responsive and modern CSS framework.
 - [Copyright and License](#license)
 
 ## <a name="gettingstarted"></a>Getting started
-simplistika.css is available via either NPM (preferred), Yarn, or a CDN.
+simplistika.css is available via either direct installation, NPM, Yarn, or a CDN.
+
+### KISSest Installation
+
+1. Clone the file wherever you want via wget or curl
+
+```shell
+wget https://raw.githubusercontent.com/Webments/simplistika.css/master/simplistika-min.css
```

Comment:
`simplistika-min.css` should be `simplistika.min.css`

Reply:
k